packages/zwave-js/src/lib/serialapi/capability/GetSerialApiInitDataMessages.ts

```diff
@@ -215,3 +218,115 @@ export class GetSerialApiInitDataResponse extends Message {
 //                         is SUC:             true
 //                         chip type:          7
 //                         chip version:       0
+
+export interface GetLongRangeNodesRequestOptions extends MessageBaseOptions {
+	listStartOffset128: number;
+}
+
+@messageTypes(MessageType.Request, FunctionType.GetLongRangeNodes)
+@expectedResponse(FunctionType.GetLongRangeNodes)
+@priority(MessagePriority.Controller)
+export class GetLongRangeNodesRequest extends Message {
+	public constructor(
+		host: ZWaveHost,
+		options:
+			| MessageDeserializationOptions
+			| GetLongRangeNodesRequestOptions,
+	) {
+		super(host, options);
```

Comment:
Getting the list of nodes is part of setting up the serial API, but I could equally see this going in with the other LR messages in the new file below...